src/main.js

```diff
@@ -212,6 +216,31 @@ function showNewView() {
   saveView.classList.add("hidden")
 };
 
+//Save poster function
+
+function savePoster() {
+  var display = "";
+  for (var i = 0; i < savedPosters.length; i++) {
+    display += `<article class="mini-poster">
+      <img class="poster-img" id=${savedPosters[i].id} src="${savedPosters[i].imageURL}" alt="nothin' to see here">
+      <h2 class="poster-title">${savedPosters[i].title}</h2>
+      <h4 class="poster-quote">${savedPosters[i].quote}</h4>
+    </article>`
+  }
+  savedPosterGrid.innerHTML = display;
+};
+
+function avoidDuplicates() {
```

Comment:
Could this function be named differently to better describe what it does? Appears to save unique items but not clear from where it is called (as event handler function for save button click event)

Reply:
I see what you mean Jon. I've updated this function name to be more accurate.